io-package.json

```diff
@@ -1,7 +1,7 @@
 {
   "common": {
     "name": "shelly",
-    "version": "8.1.1",
+    "version": "8.1.2",
```

Comment:
Versions will be updated by the release script. Don't change them manually

Reply:
Thank you for the info. Should I change this and re-create the PR?

I hope I have integrated it correctly, as I haven't done this very often.

Reply:
just push a commit with this changed back